contracts/zero-ex/contracts/src/features/UniswapFeature.sol

```diff
@@ -43,6 +43,11 @@ contract UniswapFeature is
     /// @dev AllowanceTarget instance.
     IAllowanceTarget private immutable ALLOWANCE_TARGET;
 
+    // Maximum amount of gas to forward to the initial transferFrom() call.
+    // We cap the gas for tokens that throw rather than revert, which would
+    // otherwise consume virtually all remaining gas, preventing us from falling
+    // through to the old allowance target.
+    uint256 constant private TRANSFER_FROM_GAS_LIMIT = 200e3;
```

Comment:
200k sound reasonable to everyone?

Reply:
The list is way longer than that so it's impractical to be comprehensive. Maybe we can special case the top 2?

Reply:
this might just be the scenario @mzhu25 has been waiting his entire life for with bloom filters :wink: